dashboard/app/models/pd/application/application_base.rb

```diff
@@ -49,69 +49,24 @@ class ApplicationBase < ApplicationRecord
 
     acts_as_paranoid # Use deleted_at column instead of deleting rows.
 
-    OTHER = 'Other'.freeze
-    OTHER_WITH_TEXT = 'Other:'.freeze
-    YES = 'Yes'.freeze
-    NO = 'No'.freeze
-    NONE = 'None'.freeze
-    INCOMPLETE = 'Incomplete'.freeze
-
-    COMMON_OPTIONS = {
-      title: %w(Mr. Mrs. Ms. Mx. Dr.),
-
-      state: get_all_states_with_dc.to_h.values,
-
-      gender_identity: [
-        'Female',
-        'Male',
-        'Non-binary',
-        'Preferred term not listed',
-        'Prefer not to answer'
-      ],
-
-      race: [
-        'White',
-        'Black or African American',
-        'Hispanic or Latino',
-        'Asian',
-        'Native Hawaiian or other Pacific Islander',
-        'American Indian/Alaska Native',
-        OTHER,
-        'Prefer not to answer'
-      ],
-
-      course_hours_per_year: [
-        'At least 100 course hours',
-        '50 to 99 course hours',
-        'Less than 50 course hours'
-      ],
-
-      terms_per_year: [
-        '1 quarter',
-        '1 trimester',
-        '1 semester',
-        '2 trimesters',
-        'A full year',
-        OTHER_WITH_TEXT
-      ],
-
-      school_type: [
-        'Public school',
-        'Private school',
-        'Charter school',
-        'Other'
-      ]
-    }
-
     has_many :emails, class_name: 'Pd::Application::Email', foreign_key: 'pd_application_id'
     has_and_belongs_to_many :tags, class_name: 'Pd::Application::Tag', foreign_key: 'pd_application_id', association_foreign_key: 'pd_application_tag_id'
+    belongs_to :user
+    belongs_to :regional_partner
 
-    after_initialize -> {self.status = :unreviewed}, if: :new_record?
-    before_create -> {self.status = :unreviewed}
-    after_initialize :set_type_and_year
     before_validation :set_type_and_year
+    before_validation -> {self.status = (sanitize_form_data_hash && sanitize_form_data_hash[:status] || :unreviewed)},
+      if: -> {form_data_changed? || new_record?}
```

Comment:
As for tests, there was already a test for the default status being unreviewed. I added a test "form data overwrites default status" ... so also add a test that we can manually update the form's status? I just added some refactors to other tests that were failing because of this problem––thoughts?

Reply:
sounds good to me on both counts. thanks!